apps/provisioning_api/lib/Controller/AUserData.php

```diff
@@ -110,6 +110,13 @@ public function __construct(string $appName,
 	 * @throws OCSNotFoundException
 	 */
 	protected function getUserData(string $userId, bool $includeScopes = false): ?array {
+		\OC::$server->get(\Psr\Log\LoggerInterface::class)->error(
```

Comment:
```suggestion
		\OCP\Server::get(\Psr\Log\LoggerInterface::class)->error(
```
Same for other occurences.